src/content/docs/blog/2025/07/11/Development update: os-tools/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Development update: os-tools"
+date: 2025-07-11T12:18:00Z
+authors: [nomadiccore,ermo]
+tags: [news]
+---
+
+In our recent mid-year blog post, we mentioned that it would be the first in a short series of posts providing updates on the various work streams we have been actively progressing during the last few months. Whilst that post focused primarily on our infrastructure, in this one, we will shift our focus towards the work we have been doing around our [os-tools](https://github.com/AerynOS/os-tools).
+
+## Short overview
+
+To recap, our os-tools consist of [Moss](https://aerynos.com/blog/2023/12/19/end-of-year-summary/#moss) and [Boulder](https://aerynos.com/blog/2023/12/19/end-of-year-summary/#boulder). Whilst also originally written in DLang, initial ports of these were built in Rust during the latter half of 2023. Though we made the odd improvement here and there during 2024, through q2 2025, we set out to review the code, develop an improvement plan and then put that into action.
+
+The **TL;DR** is that for:
+
+- **Moss:** Existing PRs reviewed, refined and merged (including a PR enabling faster package installation via parallel blitting). The code received various bug fixes and refactors for correctness and maintainability.
+- **Boulder:** Similarly, existing PRs reviewed, refined and merged including a few smaller features and packaging macros being added. Since Boulder uses the moss crate, it now builds packages slightly quicker due to faster buildroot creation. The odd bugfix was also made.
```

Comment:
```suggestion
- **Boulder:** Similarly, existing PRs reviewed, refined and merged including a few smaller features and packaging macros being added. Since Boulder uses the `moss` crate, it now builds packages slightly quicker due to faster buildroot creation. The odd bugfix was also made.
```